src/kayenta/report/detail/graph/semiotic/timeSeries.tsx

```diff
@@ -186,25 +172,30 @@ export default class TimeSeries extends React.Component<ISemioticChartProps, ITi
       };
     });
 
-    const millisSetBaselineAll = intDataSet.map((ds: IInterimDataSet) => ds.timestampMillisBaseline);
+    const millisSetNormalizedMinimap: number[] = dataSets[0].coordinatesUnfiltered.map(
+      (c: any) => c.timestampMillisNormalizedMinimap,
+    );
 
-    // For the main line chart, filter by brush if applicable
-    const intDataSetMain = !userBrushExtent
-      ? intDataSet
-      : intDataSet.filter((ds: IInterimDataSet) => {
+    const millisSetNormalizedMain: number[] = (!userBrushExtent
+      ? dataSets[0].coordinatesUnfiltered
+      : dataSets[0].coordinatesUnfiltered.filter((c: IDataPoint) => {
           return (
-            ds.timestampMillisBaseline >= userBrushExtent[0].valueOf() &&
-            ds.timestampMillisBaseline <= userBrushExtent[1].valueOf()
+            !userBrushExtent ||
```

Comment:
haven't we already confirmed that `userBrushExtent` exists here, since we're in the 'else' block of the ternary?

Reply:
Good catch. That's a remnant of the prev code that forgot to remove.